app/reducers/events.ts

```diff
@@ -309,8 +309,9 @@ export const selectAllRegistrationsForEvent = createSelector(
   (state, props) => props.eventId,
   (registrationsById, registrationItems, usersById, eventId) =>
     registrationItems
-      .map((regId, i) => {
-        const registration = registrationsById[regId];
+      .map((regId) => registrationsById[regId])
```

Comment:
I might be dumb, but is this map really needed now that you map on registration after the filter? Is it not the equivalent to filter directly on `registrationItems`?

Reply:
Yeah, you are right, not fully necessary. The reason I added it was to avoid doing `registrationsById[regId]` twice, since that is more verbose and maybe harder to understand. Happy to switch to using `registrationsById[regId]` if you prefer that